hr_timesheet_task_required/views/res_config_settings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="hr_timesheet.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//block[@name='project_time']" position="inside">
+                <setting
+                    id="is_timesheet_task_required_setting"
+                    help="Require task selection on each timesheet entry by default"
+                >
+                    <field name="is_timesheet_task_required" />
+                </setting>
+            </xpath>
+        </field>
```

Comment:
I think the Settings UI is not correct: 
![image](https://github.com/user-attachments/assets/c3583150-dded-4f50-8caa-da48c3352d7a)

in version 17 it looked like this: 
![image](https://github.com/user-attachments/assets/71e9dde9-124f-4514-96fd-d97415e7d0d3)